src/Carbon/Carbon.php

```diff
@@ -240,17 +242,19 @@ public function __construct($time = null, $tz = null)
     /**
      * Create a Carbon instance from a DateTime one.
      *
-     * @param \DateTime $dt
+     * @param \DateTime|\DateTimeImmutable|\DateTimeInterface $dt
      *
      * @return static
      */
-    public static function instance(DateTime $dt)
+    public static function instance($dt)
```

Comment:
Indeeed https://github.com/lucasmichot/Carbon/commit/910028cdf4879dea0a718f8c6157ee370f8e1e9f is failing...
I think it will be safe to wait 2.0